envfile/envfile.go

```diff
@@ -0,0 +1,107 @@
+package envfile
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/bitrise-io/bitrise/v2/log"
+	"github.com/bitrise-io/colorstring"
+	envmanModels "github.com/bitrise-io/envman/v2/models"
+	"gopkg.in/yaml.v3"
+)
+
+// DefaultEnvfilePathEnv is the env var that points to the (platform-dependent) envfile location.
+const DefaultEnvfilePathEnv = "BITRISEIO_ENVFILE_PATH"
+
+type EnvFile struct {
+	Envs       map[string]string `yaml:"envs"`
+	ErasedEnvs []string          `yaml:"erased_envs"`
+}
+
+// GetEnv returns the true value of an env var, even if it was not exposed to the CLI process because of its size.
+// Typical large env vars are git-related build trigger env vars, like BITRISE_GIT_COMMIT_MESSAGES or BITRISE_GIT_CHANGED_FILES.
+// If these were exposed as env vars to the CLI process, the execve() syscall would fail because it has a limit on
+// the size of all env vars and arguments. Instead, the agent launching the Bitrise CLI process clears these env vars and
+// stores their original values in a file on disk.
+// Why is this whole thing not implemented in envman? Because (currently) Bitrise CLI interacts with envman through
```

Comment:
Are we sure about this part? We tried to clear envman (and stepman) binary calls. As far as I see we use envman as a Go package.
Although this doesn't really matter for this change, so we might just remove this part of the comment.